scripts/verify-phase-52-1-cli-command-contract.sh

```
@@ -0,0 +1,138 @@
+  "## 6. Failure and Retry Rules"
+  "## 7. Forbidden Claims"
+  "## 8. Validation"
+  "## 9. Non-Goals"
+)
+
+required_phrases=(
+  "- **Status**: Accepted"
+  "- **Date**: 2026-05-01"
+  "- **Related Issues**: #1063, #1064"
+  "This contract defines the first-user CLI command contract only. It does not implement installer behavior, Wazuh profile generation, Shuffle profile generation, first-user UI flows, AI operations, SIEM breadth, SOAR breadth, packaging, release-candidate behavior, or general-availability behavior."
+  'The approved command names are `init`, `up`, `doctor`, `seed-demo`, `status`, `open`, `logs`, and `down`.'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)